src/main/java/org/folio/circulation/domain/policy/CirculationPolicyRepository.java

```diff
@@ -70,8 +75,8 @@
       .thenApply(result -> result.next(this::mapToPolicy));
   }
 
-  public CompletableFuture<Result<String>> lookupPolicyId(Item item, User user) {
-    CompletableFuture<Result<String>> findLoanPolicyCompleted = new CompletableFuture<>();
+  public CompletableFuture<Result<Pair<String, Integer>>> lookupPolicyId(Item item, User user) {
+    CompletableFuture<Result<Pair<String, Integer>>> findLoanPolicyCompleted = new CompletableFuture<>();
```

Comment:
I would prefer this used a type that was more expressive, how does a client know what the string and integer are?

Reply:
this code was outdated for now

Reply:
Does that mean that the comment no longer applies?